apps/mobile/src/app/(home)/home/home.tsx

```diff
@@ -1,12 +1,45 @@
+import { useEffect, useRef } from 'react';
+
+import { NotificationsSheet } from '@/components/notifications/notifications-sheet';
+import { useRegisterNotifications } from '@/components/notifications/use-notifications';
 import { PageLayout } from '@/components/page/page.layout';
 import { AccountsWidget } from '@/components/widgets/accounts/accounts-widget';
 import { AllAccountBalances, TokensWidget } from '@/components/widgets/tokens/tokens-widget';
+import { useSettings } from '@/store/settings/settings';
 import { useWallets } from '@/store/wallets/wallets.read';
+import { featureFlagKeys } from '@/utils/feature-flag';
+import { useBoolVariation } from '@launchdarkly/react-native-client-sdk';
 import { useLingui } from '@lingui/react';
 
+import { SheetRef } from '@leather.io/ui/native';
+
 export function Home() {
   useLingui();
   const { hasWallets } = useWallets();
+  const notificationSheetRef = useRef<SheetRef>(null);
```

Comment:
Do you think this could be worth abstracting into one / two hooks that encapsulate this logic?

Reply:
eg. `useConfigureNotifications(sheetRef)`

Reply:
👍 to this and for moving it to `features/notifications` along with the other notifications code